packages/manager/modules/pci/src/projects/project/private-registry/private-registry.html

```diff
@@ -75,6 +75,11 @@
                 data-compact
                 data-ng-if="$row.status !== $ctrl.PRIVATE_REGISTRY_STATUS.DELETING"
             >
+                <oui-action-menu-item
+                    data-on-click="$ctrl.upgradePlan($row.id)"
```

Comment:
If I remember correctly, there are conditions for this button to be available. Not all registries can be upgraded (https://github.com/ovh/manager/pull/2173/commits/a910e917235c9172618115d5d9beafe73ec9b8b1#diff-81a1ecaa10007b23ab1c29805e61fe18R11)

Reply:
Right now i have added a disable if the plan is already Large, I will check on this one

Reply:
Thats the only check for now it seems, got it confirmed.